haystack/components/generators/chat/hugging_face_tgi.py

```diff
@@ -8,9 +8,13 @@
 from haystack.utils import Secret, deserialize_callable, deserialize_secrets_inplace, serialize_callable
 from haystack.utils.hf import HFModelType, check_generation_params, check_valid_model, list_inference_deployed_models
 
-with LazyImport(message="Run 'pip install transformers'") as transformers_import:
-    from huggingface_hub import InferenceClient
-    from huggingface_hub.inference._text_generation import TextGenerationResponse, TextGenerationStreamResponse, Token
+with LazyImport(message="Run 'pip install \"huggingface_hub>=0.22.0\" transformers'") as transformers_import:
```

Comment:
This might seem redundant since `huggingface_hub` is a dependency of `transformers`, but is done to enforce our encouragement.
(Furthermore, in the future I want to remove the `transformers` dependency for this component)

Reply:
In which cases will `pip install huggingface_hub` pull down older version if executed by some user in the future? Perhaps we are mentioning this version specifically just to make sure and be explicit if user somehow installed older version of transformers and with it older huggingface_hub, right?